content/tutorials/spatial_wfs_services/index.md

```diff
@@ -423,6 +423,21 @@ The map of regions of Belgium.
 
 ``` r
 wfs_regions <- "https://eservices.minfin.fgov.be/arcgis/services/R2C/Regions/MapServer/WFSServer"
+regions_client <- WFSClient$new(wfs_regions, 
+                            serviceVersion = "2.0.0")
+```
+
+    ## Warning in CPL_crs_from_input(x): GDAL Message 1: +init=epsg:XXXX syntax is
+    ## deprecated. It might return a CRS with a non-EPSG compliant axis order.
```

Comment:
@hansvancalster  Maybe this cannot be avoided by you, but it seems that somewhere in the pipeline (non-GDAL3-compliant) proj4strings have been transmitted (it should be avoided, cf https://www.r-spatial.org/r/2020/03/17/wkt.html and e.g. https://geocompr.github.io/presentations/). I wonder whether you can identify where it comes from - in that case it would be a good idea to warn the involved developers about this.

Reply:
I filed an issue in ows4R package: https://github.com/eblondel/ows4R/issues/47